docs/reference/toc.yaml

```diff
@@ -109,6 +109,8 @@ items:
         href: toloka.client.TolokaClient.get_operation_log.md
       - name: wait_operation
         href: toloka.client.TolokaClient.wait_operation.md
+      - name: wait_operation_with_progress
```

Comment:
Please don't forget to remove it here if you'll implement the feature with tqdm's `disable` parameter (see other comment).